### DISCUSSION TITLE
Ubuntu Codenames - disable update

### DISCUSSION THREAD

**Discussion by esmadau** [renovatebot/renovate#23415] [Suggest an Idea]
### Tell us more.

We would like to be able to disable codename updates such as
- FROM ubuntu:focal
+ FROM ubuntu:jammy

but continue getting updates within focal

**Comment by viceice** [comment]
use a package rule with match current value and enable false

**Comment by esmadau** [comment]
If anyone else comes here, I did this:
    {
      "enabled": false,
      "matchDatasources": [
        "docker"
      ],
      "matchCurrentValue": "ubuntu:focal",
      "matchFileNames": ["docker/foo/Dockerfile"]
    }

**Reply by viceice** [reply]
please format the code as codeblock
